qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/ClusteringRule.java

```diff
@@ -283,12 +283,13 @@ private Broker createBroker(final int nodeId) {
         new SystemContext(brokerCfg, brokerBase.getAbsolutePath(), controlledClock);
     systemContexts.put(nodeId, systemContext);
 
+    systemContext.getScheduler().start();
+
     final Broker broker = new Broker(systemContext, getSpringBrokerBridge(nodeId));
 
     broker.addPartitionListener(new LeaderListener(partitionLatch, nodeId));
     new Thread(
             () -> {
-              systemContext.getScheduler().start();
```

Comment:
Previously it was working fine even if the actor was submitted before the scheduler is started? :thinking: It seems the task is not lost, but eventually get picked up once the scheduler is started.

Reply:
Seems like it, yes